common/riff.ksy

```diff
@@ -0,0 +1,93 @@
+meta:
+  id: riff
+  title: Resource Interchange File Format (RIFF)
+  license: CC0-1.0
+  endian: le
+  encoding: ASCII
+  xref:
+    justsolve: RIFF
+    loc: fdd000025
+    wikidata: Q1196805
+doc: |
+  The Resource Interchange File Format (RIFF) is a generic file container format
+  for storing data in tagged chunks. It is primarily used to store multimedia
+  such as sound and video, though it may also be used to store any arbitrary data.
+
+  The Microsoft implementation is mostly known through container formats
+  like AVI, ANI and WAV, which use RIFF as their basis.
+doc-ref: https://www.johnloomis.org/cpe102/asgn/asgn1/riff.html
+seq:
+  - id: riff_chunk
+    type: chunk
+types:
+  parent_chunk_data:
+    seq:
+      - id: form_type
+        type: str
+        size: 4
+        pad-right: 0x20
```

Comment:
Not sure that we need this `pad-right`. `pad-right` is the contract that all the strings have padding stripped here. But the strings are 4 bytes, 4 bytes is 1 dword, one dword is a unit for reading memory on 32-bit archs and is faster than reading 3 bytes. The strings are fixed length. There is no chance that there will be arbitrary long padding. So IMHO `pad-right` here doesn't make any sense and would have prevened KSC from optimizing the code if such optimizations ever were introduced.